src/azure-cli/azure/cli/command_modules/vm/aaz/latest/network/nic/_list.py

```diff
@@ -48,12 +48,12 @@ def _build_arguments_schema(cls, *args, **kwargs):
 
     def _execute_operations(self):
         self.pre_operations()
-        condition_0 = has_value(self.ctx.subscription_id) and has_value(self.ctx.args.resource_group) is not True
-        condition_1 = has_value(self.ctx.args.resource_group) and has_value(self.ctx.subscription_id)
+        condition_0 = has_value(self.ctx.args.resource_group) and has_value(self.ctx.subscription_id)
+        condition_1 = has_value(self.ctx.subscription_id) and has_value(self.ctx.args.resource_group) is not True
```

Comment:
Why variable name and method sequence are swapped?

Reply:
Because the command mode changed. We will fix this problem in the upcoming aaz-dev-tools